assets/src/edit-story/components/form/switch.js

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
I'd like to see both a unit test (testing functionality, not implementation) and a story (for Storybook) for this component.

Reply:
Story and unit testing added.